packages/dsh-tauri/src/client/index.ts

```
@@ -23,6 +23,8 @@ export * from './storage'
 export * from './store'
 export type * from './types'
 
+export { invokeBridgedTauri } from './service/invoke'
```

ESLint

[error] Expected "./service/invoke" to come before "./types".

(perfectionist/sort-exports)